kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -169,6 +169,14 @@
   public static final ConfigDef.Range PRODUCE_GRACE_PERIOD_MS_VALIDATOR =
       ConfigDef.Range.between(0, Integer.MAX_VALUE);
 
+  public static final String PRODUCE_EXECUTOR_THREAD_POOL_SIZE =
+      "api.v3.produce.executor.thread.pool.size";
+  private static final String PRODUCE_EXECUTOR_THREAD_POOL_SIZE_DOC =
+      "Number of threads in the executor thread pool used to process produce responses.";
+  public static final String PRODUCE_EXECUTOR_THREAD_POOL_SIZE_DEFAULT = "50";
+  public static final ConfigDef.Range PRODUCE_EXECUTOR_THREAD_POOL_SIZE_VALIDATOR =
+      ConfigDef.Range.between(1, Integer.MAX_VALUE);
```

Comment:
Instead of being exclusive for produce, I would just call this non-blocking-thread-pool, or something like that. It is useful even for Admin (which we should also fix eventually). So, `non.blocking.thread.pool.size`, or something like that.

Reply:
What do you mean by non-blocking? This ends up blocking on `writev` syscalls - the way I understand it, this is a big part of why we are moving it away from the Kafka threads.

Reply:
Yes. I forgot the very last future will be blocking. Let's keep it separate then just for produce. +1 for `produce.response` or `produce.callback` or something like that.

Reply:
The blocking doesn't affect whether we use this for admin calls or produce calls really though - we just want to not get tin the way of the kafka processing.

I've gone with `api.v3.rest.callback.thread.pool.size` but can easily change it if there are concerns

Reply:
(And in the end I didn't, based on another of Dimitar's comments)